### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by mnonnenmacher**
Some preparations for the next release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 - First public release published under the Apache-2.0 license
 - Rename plugin to gradle-job-dsl-plugin
+- Replace deprecated JobBuilder with JobBuilder2
```
Comment (sschuberth) [inline CHANGELOG.md]: Maybe better say "Rename JobBuilder2 to JobBuilder, replacing the original JobBuilder" as JobBuilder2 is not named anymore like that?

File: README.md [hunk 2]
```diff
@@ -98,46 +98,48 @@ those classes in your Job DSL scripts, because they are automatically added to t
 
 The recommended way to implement job templates are job builder classes. The plugin provides a basic `JobBuilder` class
 that can be extended to create templates. Below you can find an example of how to create a template that automatically
-enables the timestamp plugin for all your jobs:
+enables the timestamp plugin for all jobs using it:
```
Comment (sschuberth) [inline README.md]: "for all jobs that use the template" would be clearer, IMO.

**Comment by sschuberth** [comment]
Nit: 264ba9333716223eccd14e84f4edc6640aeae5c8 says that "JobBuilder2 will be renamed to
JobBuilder in the next commit", but in fact it's the commit after the next commit. Maybe just say "in one of the next commits"?

File: CHANGELOG.md (hunk 1 above)
Comment (mnonnenmacher) [inline CHANGELOG.md]: Done

File: README.md (hunk 2 above)
Comment (mnonnenmacher) [inline README.md]: Done

**Comment by mnonnenmacher** [comment]
@sschuberth Regarding the commit message, actually it is the next commit.

**Pull request merged**